app/lib/service/entrypoint/frontend.dart

```diff
@@ -63,7 +63,7 @@ Future<void> watchForResourceChanges() async {
   void setupWatcher(
       String name, String path, FutureOr<void> Function() updateFn) {
     final w = Watcher(path, pollingDelay: Duration(seconds: 3));
-    final subs = w.events.debounce(Duration(milliseconds: 200)).listen(
+    final subs = w.events.debounce(Duration(milliseconds: 2500)).listen(
```

Comment:
Is this related?

Reply:
Yes, although only in development mode with `--watch` flag: without it on my machine the rebuild got triggered way too fast, often failing with syntax error due to unfinished line being saved by the IDE. A slightly longer time provides much better workflow.